### ISSUE TITLE
[Request] Add "print-hide/only" helper classes

### ISSUE THREAD

**Issue by sebastienroul** [quasarframework/quasar#950]
It would be very nice to add a CSS class "print-hide" to remove the element when the user print the page, like "desktop-hide" or "within-iframe-hide"

Best regards

**Comment by sebastienroul** [comment]
Workaround for me, at this time is adding : 
```
@media print {
        .print-hide, aside {
          display:none !important;
        }
        .layout-page-container {
          padding: 0px !important;
        }
      }
```
I have to set `aside` cause just the print-hide on `div slot="left" ` was not enought...
And the `pdding` on `.layout-page-container` is to extend it in full page...

**Comment by rstoenescu** [comment]
`print-hide` and `print-only` helper classes are available in edge and future v0.14.5.

I avoided also adding the CSS for aside and layout-page-container since someone might actually want that printed out. If I add CSS code for it, it will be gone.

Care for a PR on docs with your tip though? Thanks for suggesting this!

**Comment by mstaack** [comment]
docs update: https://github.com/quasarframework/quasar-framework.org/pull/270
